src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -32,7 +35,7 @@ public override void Initialize(IHostContext hostContext)
             _dockerManger = HostContext.GetService<IDockerCommandManager>();
         }
 
-        public async Task StartContainerAsync(IExecutionContext executionContext, object data)
+        private async Task StartContainerAsync(IExecutionContext executionContext, object data)
```

Comment:
should we do all version check in `StartContainersAsync()`? so we don't check docker version 100 times. :)

Reply:
Done